modules/microvm/virtualization/microvm/guivm.nix

```diff
@@ -252,6 +252,21 @@ let
           # We dont enable services.blueman because it adds blueman desktop entry
           services.dbus.packages = [ pkgs.blueman ];
           systemd.packages = [ pkgs.blueman ];
+
+          systemd.user.services.audio-control = {
+            enable = true;
+            description = "Audio Control application";
+
+            serviceConfig = {
+              Type = "simple";
+              Restart = "always";
+              RestartSec = "1";
+              ExecStart = "${pkgs.ghaf-audio-control}/bin/GhafAudioControlStandalone --pulseaudio_server=audio-vm:4713 --deamon_mode true";
```

Comment:
The convention used to provide options and their arguments is inconsistent - pulseaudio server argument is provided with '=', daemon mode argument is separated with a space.
Ref. https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html

Also `--daemon_mode true` seems a bit redundant - it might be a more user-friendly approach to use it without a value (--daemon) and treat its presence as "true."

On a different note, pulseaudio TCP control port shouldn't be hard-coded anymore.
We can now use something like this:
```
let
  ...
  inherit (config.ghaf.services.audio) pulseaudioTcpControlPort;
  ...
in
  ...
  ExecStart = "${pkgs.ghaf-audio-control}/bin/GhafAudioControlStandalone --pulseaudio_server=audio-vm:${toString pulseaudioTcpControlPort}
  ...
```